Khan/perseus
# Add mafs flag to editor page storybook demo

## Summary:
Adding mafs flag to the storybook for EditorPage so that it's easier
to test changes to the interactive graph editor + mafs with the
full context, layout, and styling.

Issue: none

## Test plan:
- Go to http://localhost:6006/?path=/docs/perseus-editor-editorpage--docs
- Click on "add a widget..."
- Click interactive graph
- open the "interactive graph 1" accordion section
- Change the type of graph to "line segment(s)"
- Confirm that a mafs graph shows up along with the "Add element" select at the bottom

<img width="1462" alt="Screenshot 2024-04-01 at 1 41 14 PM" src="https://github.com/Khan/perseus/assets/13231763/1260f544-b3bc-4280-87db-6ef7e4393387">
